src/Tizen.Multimedia/Common/IWindowProvider.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+
+namespace Tizen.Multimedia
+{
+    /// <summary>
+    /// The IWindowProvider interface provides the window handle and information about the window's position, size, and rotation.
+    /// </summary>
+    /// <since_tizen> 12 </since_tizen>
+    public interface IWindowProvider
+    {
+        /// <summary>
+        /// Gets the window handle
+        /// </summary>
+        /// <remarks>
+        /// This handle represents Ecore_Wl2_Window on native.
+        /// </remarks>
+        IntPtr WindowHandle { get; }
```

Comment:
It's better to avoid IntPtr in public API, but I understand benefits and that this is an acceptable case. It is in fact not to be used by application.